install-dependencies.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o nounset -o errexit
+
+MY_MODULES="dagrt leap loopy meshmode grudge mirgecom"
+
+for m in $MY_MODULES; do
+    if [[ ! -f $m/setup.py ]]; then
+        echo "==== ERROR: incomplete git clone. Please run:"
+        echo "====   git submodule init && git submodule update"
+        echo "==== to fetch emirge's submodules."
+        exit 1
+    fi
+done
+
+echo "==== Installing Conda packages"
+
+conda update --all --yes
```

Comment:
Can the `conda update` be left out, or is it needed here?

Reply:
I think it's better to leave it here. Even when downloading the latest Miniforge install script, there are always updates immediately available.

Reply:
Could we maybe put it in `install-conda.sh`? Just thinking that if I'm installing emirge into an existing environment, I may not want to update every package that I have (just whatever is needed for mirgecom to work).

Reply:
Good point. Fixed in the most recent commit.